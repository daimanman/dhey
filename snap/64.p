{"C":1,"Duration":44887828,"EndTime":1537175258528084500,"Err":"","FileId":"3","InputFileName":"file","Method":"POST","N":1,"P":null,"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1537175258483196700,"Status":1,"TaskId":64,"Type":"FILE","Url":"http://127.0.0.1:9091/api/v1/oss","Z":""}