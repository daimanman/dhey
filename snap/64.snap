{"AvgTotal":0.0149694,"Fastest":0.0149694,"Slowest":0.0149694,"Average":0.0149694,"Rps":66.80294467380122,"AvgConn":0,"AvgDNS":0,"AvgReq":0,"AvgRes":0.0010058,"AvgDelay":0.0079786,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0010058,"DelayMax":0.0079786,"DelayMin":0.0079786,"Lats":[0.0149694],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0.0010058],"DelayLats":[0.0079786],"Total":14969400,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":16,"SizeReq":16,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0149694,"Count":1,"Frequentcy":1},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0},{"Mark":0.0149694,"Count":0,"Frequentcy":0}],"TaskId":64,"Duration":0.0149694}